packages/web-react/src/components/Modal/ModalHeader.tsx

```diff
@@ -6,8 +6,13 @@ import ModalCloseButton from './ModalCloseButton';
 import { useModalContext } from './ModalContext';
 import { useModalStyleProps } from './useModalStyleProps';
 
+const defaultProps: ModalHeaderProps = {
+  hideCloseButton: false,
```

Comment:
Hmm, I have to say I'm a little bit confused when you use prefix [`has`/`is`](https://github.com/lmc-eu/spirit-design-system/pull/1533#discussion_r1664003667) and where not.

Reply:
🤔 When the prop is of type Boolean the `is/has` prop should be used to indicate this and thus be more readable. In some cases, we also use `enable` prefix for some options. There exists `hideOnCollapse` on the Accordion component also. In this case, I am not sure if we want to create the easy-to-use API `<ModalHeader hideCloseButton />` or something worse usable to suggest that this is not a good idea like `<ModalHeader hasCloseButton={false} />`.

Reply:
Discussed with @crishpeen: `hasCloseButton` is good enough but we should talk about this topic during Great (Long) Topics in the future.

Reply:
prop renamed to to `hasCloseButton` 👍